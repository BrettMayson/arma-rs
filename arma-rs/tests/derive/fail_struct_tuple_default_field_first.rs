use arma_rs::{FromArma, IntoArma};

#[derive(FromArma, IntoArma)]
struct DeriveTest(#[arma(default)] String, u32, bool);

fn main() {}
