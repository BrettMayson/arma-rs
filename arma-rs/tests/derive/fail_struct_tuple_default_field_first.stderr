error: field must have #[arma(default)] because previous field 0 has #[arma(default)]
 --> tests/derive/fail_struct_tuple_default_field_first.rs:4:44
  |
4 | struct DeriveTest(#[arma(default)] String, u32, bool);
  |                                            ^^^

error: field must have #[arma(default)] because previous field 0 has #[arma(default)]
 --> tests/derive/fail_struct_tuple_default_field_first.rs:4:49
  |
4 | struct DeriveTest(#[arma(default)] String, u32, bool);
  |                                                 ^^^^
